config/routes.rb

```diff
@@ -0,0 +1,21 @@
+Rails.application.routes.draw do
+  get 'root/index'
+  get 'messages/index'
```

Comment:
- [ ] [Optional] It is advisable to avoid including unnecessary files and configurations in the project. Therefore, I recommend removing the route and controller (`app/controllers/messages_controller.rb`) for messages/index.